.github/workflows/deploy.yml

```diff
@@ -37,6 +37,11 @@ on:
         default: "@pleo-io"
         description: "Org scope for the GitHub Package Registry"
         type: string
+      is_translation_files:
```

Comment:
What is the benefit to coupling the SPA deployment code with the translation file deploy? Could/should it be another workflow?